wait/metrics.go

```diff
@@ -8,10 +8,15 @@ import (
 	"net/http"
 	"time"
 
+	dto "github.com/prometheus/client_model/go"
 	"github.com/prometheus/common/expfmt"
 )
 
-func getCounter(url string, family string, labelKey string, labelValue string) (float64, error) {
+func getMetricValue(url string, family string, expectedLabels []string) (float64, error) {
+	if len(expectedLabels)%2 != 0 {
+		return -1, fmt.Errorf("Received odd number of label arguments, labels must be key-value pairs")
```

Comment:
it's just a detail, but an error string should not be capitalized
```suggestion
		return -1, fmt.Errorf("received odd number of label arguments, labels must be key-value pairs")
```

Reply:
Done in d1ffef3bfd12c67c8135e89acaeedfca3cc0206a